docs/changelog/90124.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 90142
```

Comment:
We can delete this file since this isn't something we need to include in release notes.